mediaflow_proxy/extractors/vidoza.py

```
@@ -0,0 +1,59 @@
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.mediaflow_endpoint = "video_proxy"   
+
+    async def extract(self, url: str) -> Dict[str, Any]:
+        parsed = urlparse(url)
+
+        # Accept vidoza.net / vidoza.co / videzz.net
+        valid_domains = ("vidoza.net", "vidoza.co", "videzz.net")
+        if not parsed.hostname or not (parsed.hostname in valid_domains or any(parsed.hostname.endswith(f".{d}") for d in valid_domains)):
+            raise ExtractorError("Vidoza: Invalid domain")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)